Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
```

Comment:
I've pushed the current WIP so people can keep tabs on what I'm doing, and hopefully we can work out what I've broken :wink: